pages/myProfile/profilePage.module.scss

```diff
@@ -0,0 +1,6 @@
+.userPageRoot {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
`align-items: center` is unnecessary in principle. In addition, since some of the content in the right column may be larger than the avatar height, the avatar will be shifted down, which is not expected.